beast/observationmodel/ast/make_ast_input_list.py

```diff
@@ -432,34 +441,53 @@ def supplement_ast(
         print(
             "{} exists. Will attempt to load SEDs for ASTs from there \
             and remove those SEDs from the SED grid".format(
-                existingASTfile))
+                existingASTfile
+            )
+        )
 
-        t = Table.read(existingASTfile, format="fits")
-        sedsMags = np.delete(sedsMags, t["sedgrid_indx"], axis=0)
-        sedsIndx = np.delete(sedsIndx, t["sedgrid_indx"])
+        t = Table.read(existingASTfile, format="ascii")
```

Comment:
If you are going to decide not to save sedgrid_indx in the ascii file, then this part needs to be reverted.